pkg/operator/controller/nodeobservability/daemonset.go

```diff
@@ -110,7 +110,7 @@ func (r *NodeObservabilityReconciler) desiredDaemonSet(nodeObs *v1alpha1.NodeObs
 					Containers: []corev1.Container{
 						{
 							Image:           nodeObs.Spec.Image,
-							ImagePullPolicy: corev1.PullIfNotPresent,
+							ImagePullPolicy: corev1.PullAlways,
```

Comment:
Should we change the policy?

Reply:
changed this for testing - so I have reverted back to PullIfNotPresent